### PR TITLE
292  unit tests for service package

### PR DESCRIPTION
**Pull request by urio999**

### CODE REVIEW

**Review by staaankey — APPROVED** [review]

File: service-api/src/main/java/greencity/dto/search/SearchResponseDto.java [hunk 1]
```diff
@@ -1,16 +1,44 @@
 package greencity.dto.search;
 
+//import lombok.Builder;
```
Comment (DenysLiubchenko) [inline service-api/src/main/java/greencity/dto/search/SearchResponseDto.java]: Delete this comment

**Review by DenysLiubchenko — CHANGES_REQUESTED** [review]
I think we dont need to commit 13000 lines of htmlReport

File: service-api/src/main/java/greencity/dto/search/SearchResponseDto.java (hunk 1 above)
Comment (urio999) [reply service-api/src/main/java/greencity/dto/search/SearchResponseDto.java]: Commented code deleted.
Unnecessary html files are also removed.

**Review by staaankey — APPROVED** [review]

**Review by OleksandrKobetsDev — APPROVED** [review]
well done

**Review by DenysLiubchenko — APPROVED** [review]

**Pull request merged**